langserver/references.go

```diff
@@ -212,6 +212,32 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 		t := time.NewTicker(time.Second)
 		defer t.Stop()
 		streamTick = t.C
+		streamPos := 0
+		streamUpdate = func() {
+			mu.Lock()
+			partial := refs
+			mu.Unlock()
+			if len(partial) == streamPos {
+				return
+			}
+
+			patch := make([]ReferenceAddOp, 0, len(partial)-streamPos)
+			for ; streamPos < len(partial); streamPos++ {
+				patch = append(patch, ReferenceAddOp{
+					OP:    "add",
+					Path:  "/-",
```

Comment:
It's where the item should be added in the object.
See http://jsonpatch.com/ and https://tools.ietf.org/html/rfc6901 for the spec